app/Http/Requests/Auth/LoginRequest.php

```diff
@@ -45,14 +47,29 @@ public function authenticate()
     {
         $this->ensureIsNotRateLimited();
 
-        if (! Auth::attempt($this->only('email', 'password'), $this->boolean('remember'))) {
+
+        $user = User::where('email', $this->email)
+            ->orWhere('name', $this->name)
+            ->orWhere('phone', $this->phone)
+            ->first();
+
+        //dd($user);
+        //dd($user->password);
+        if (!$user || strcmp($this->password, $user->password)!=0) {
```

Comment:
Nanti Setelah Leo Update untuk Kapasitas tabelnya, boleh coba auth pake ini ya, biar engga terlalu hardcode banget compare passwordnya
Hash::check($this->password, $user->password)

Reply:
Ini bisa dipake untuk hash nya